docs/decisions/0002-events-naming-and-versioning.rst

```diff
@@ -0,0 +1,91 @@
+2. Open edX events naming and versioning
+========================================
+
+Status
+------
+
+Accepted
+
+
+Context
+-------
+
+Event type hooks are an important public promise. They are a strong foundation
+of a healthy ecosystem of extensions for the Open edX world and have been
+recognized as such by the arch team of the Open edX core and also by the community.
+
+This ADR has the purpose of defining the rules to be followed when naming an
+Open edX Event with the intent of covering the use cases of:
+
+* Open edX core developers wanting to add new events.
+* Open edX core developers wanting to deprecate and eventually remove events.
+* Open edX extension developers wanting to create and maintain stable
+  applications, even when the events framework evolves and changes over time.
+* Service developers wanting to listen for events using message queues.
+
+The decisions reached in this ADR are the product of open discussions in the ADR
+PR history.
+
+
+Decisions
+---------
+
+1. The name of an event will be a ``string`` that follows the `type format`_
+defined in the `OEP-41`_:
+
+``{Reverse DNS}.{Architecture Subdomain}.{Subject}.{Action}.{Major Version}``
```

Comment:
We can link to https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts since it is a public page.

The other pages linked in there and the google doc are private.

The oep-41 has 4/6 (https://open-edx-proposals.readthedocs.io/en/latest/oep-0041-arch-async-server-event-messaging.html#subdomain-from-domain-driven-design)